.gitignore

```diff
@@ -2,7 +2,6 @@ node_modules
 package-lock.json
 config.json
 /dist/
-.vscode/*
```

Comment:
This one ignores `.vscode` folder ;)

I guess you should just remove the 4 lines bellow.

Reply:
@Michsior14 Yep, @balloob said to ignore the settings.json file but atm that's the only one file inside that folder so why not ignore the entire folder?

Reply:
Yep, this is why I suggested to remove lines like `!.vscode/settings.json`. The above comment was added when you removed this line (`.vscode/*`) ;)

Reply:
It's there... I removed it in one commit to be able to remove the folder, then in the next commit I have readded it

Reply:
Yep I know, it's just outdated comment now.
BTW. [git rm](https://git-scm.com/docs/git-rm) is a easier way to do it ;)

Reply:
@Michsior14 Didn't know about `git rm` command :laughing: